### DISCUSSION TITLE
Welcome to englishell Discussions!

### DISCUSSION THREAD

**Discussion by tzador** [tzador/englishell#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mydaysx** [comment]
just like to say AWESOME. love this little tool so much. Genius!
